qadence/noise/protocols.py

```diff
@@ -201,6 +197,10 @@ def dephasing(self, *args: Any, **kwargs: Any) -> NoiseHandler:
         self.append(NoiseHandler(NoiseProtocol.ANALOG.DEPHASING, *args, **kwargs))
         return self
 
-    def readout(self, *args: Any, **kwargs: Any) -> NoiseHandler:
-        self.append(NoiseHandler(NoiseProtocol.READOUT, *args, **kwargs))
+    def independent_readout(self, *args: Any, **kwargs: Any) -> NoiseHandler:
+        self.append(NoiseHandler(NoiseProtocol.READOUT.INDEPENDENTREADOUT, *args, **kwargs))
+        return self
+
+    def correlated_readout(self, *args: Any, **kwargs: Any) -> NoiseHandler:
```

Comment:
Maybe consider renaming these functions to readout_correlated, and readout_indipendent so that they align with how readout is specified using the protocol in the handler?